src/screens/App/screens/Dashboard/components/Filter/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+
+import './style.css'
+
+const Filter = (props) => {
+  return (
+    <form onSubmit={props.filter} className='filter-container'>
+      <select name='type' className='filter'>
+        <option disabled defaultValue> Seleccionar filtro.. </option>
```

Comment:
In all proyects, we are following the next guideline styles:
Extracts all strings of a screen into a `strings.js` file and then import this strings in each corresponding component and subComponent.

Like this.
```
screen
   --Dashboard
          ---Components
                  (components)
          --- index.js
          --- layout.js
          --- strings.js
   --BookDetail
          ---Components
                  (components)
          --- index.js
          --- layout.js
          --- strings.js
```